packages/jsii-diff/test/util.ts

```diff
@@ -12,10 +12,15 @@ export async function expectNoError(original: string, updated: string) {
 }
 
 export async function expectError(
-  error: RegExp,
+  error: RegExp | undefined,
   original: string,
   updated: string,
 ) {
+  if (error === undefined) {
+    await expectNoError(original, updated);
+    return;
```

Comment:
I reckon this is equivalent (but saves 1 promise un-wrapping, then re-wrapping):

```suggestion
    return expectNoError(original, updated);
```

Reply:
I had that originally, but if I do that then eslint starts complaining about the regular return path:

```
{
	"resource": "/Users/huijbers/Workspaces/PublicCDK/jsii/packages/jsii-diff/test/util.ts",
	"owner": "eslint",
	"code": {
		"value": "consistent-return",
		"target": {
			"$mid": 1,
			"external": "https://eslint.org/docs/rules/consistent-return",
			"path": "/docs/rules/consistent-return",
			"scheme": "https",
			"authority": "eslint.org"
		}
	},
	"severity": 8,
	"message": "Expected to return a value at the end of async function 'expectError'.",
	"source": "eslint",
	"startLineNumber": 14,
	"startColumn": 23,
	"endLineNumber": 14,
	"endColumn": 23
}
```

And putting an empty `return` at the end of the function doesn't fix it.

Rather take the path of least resistance here and it's not like it's in a hot path or anything :).

Reply:
Makes sense!